### PR TITLE
Chores/update missing short form tests #126468679

### PR DESCRIPTION
**Pull request by chrisdolendo**

### CODE REVIEW

File: spec/requests/api/v1/listings_spec.rb [hunk 1]
```diff
@@ -130,24 +130,24 @@
     expect(json['ami'].length).to eq(9)
   end
 
-  it 'gets Lottery Preferences' do
-    VCR.use_cassette('listings/lottery-preferences') do
-      get '/api/v1/listings/lottery-preferences.json'
-    end
+  # it 'gets Lottery Preferences' do
```
Comment (chrisdolendo) [inline spec/requests/api/v1/listings_spec.rb]: @kaptron this doesn't return anything anymore by salesforce and is blank on the listing page -- should I report this as a bug on Salesforce Pivotal or is this just stale code? I'm guessing the former

File: spec/requests/api/v1/short_form_spec.rb [hunk 2]
```diff
@@ -38,10 +40,71 @@
...
+      url = '/api/v1/short-form/listing-application/a0Wf0000003j03WEAQ.json'
+      VCR.use_cassette('shortform/show_application') do
+        get url, {}, @auth_headers
+      end
+      expect(response).to be_success
+    end
+  end
+
+  # describe 'submit_application' do
+  # RSPEC IS CONVERTING THE BOOLEAN TO STRINGS, SALESFORCE REJECTING PARAMS
```
Comment (chrisdolendo) [inline spec/requests/api/v1/short_form_spec.rb]: Running across this issue when I try to run this test: https://github.com/rspec/rspec-rails/issues/610

converts the boolean from the params to strings. WAI.

Comment (kaptron) [inline spec/requests/api/v1/short_form_spec.rb]: hm it does make sense that the params are converted to strings, anything you post via a request will be strings. I can take a look at it and see what you mean.

File: spec/requests/api/v1/listings_spec.rb (hunk 1 above)
Comment (kaptron) [inline spec/requests/api/v1/listings_spec.rb]: @chrisdolendo correct, this was an old endpoint that was never really properly implemented -- in fact that's one of the big changes coming down the pipe from Luke is figuring out how to query for Listing Lottery Preferences, so he said this salesforce endpoint was going to be deprecated.

**Comment by kaptron** [comment]
@chrisdolendo I couldn't figure out the VCR bool -> string issue exactly so I just setup the JSON without any booleans, created the cassette with that, and at least the test will pass. I added a note about it (basically if we want to re-create that cassette later) but at least now the test can run and pass.

**Pull request merged**